src/content/docs/en/guides/cms/payload.mdx

```diff
@@ -6,7 +6,222 @@ stub: true
+3. **A PayloadCMS REST API** - Create a [PayloadCMS](https://payloadcms.com/) project and connect it to your MongoDB database during the setup.
+
+### Installing PayloadCMS
+
+Follow the [PayloadCMS Installation docs](https://payloadcms.com/docs/getting-started/installation) in order to install a fresh PayloadCMS app. During the PayloadCMS installation, you will be asked if you want to use a template.
+
+If you pick one of the available templates (such as 'blog'), additional collections will be generated for you. You can then skip the following section, because you already have a collection other than "Users" that you can use.
+
+If you don't pick any templates, only a single collection called "Users" will be generated.
+
+If you start the PayloadCMS server using `npm run dev`, you can go to `http://localhost:3000/admin` to enter your PayloadCMS Dashboard for the first time. Create the initial administrator account and enter the Dashboard. You should now see that the "Users" collection contains just the administrator account.
+
+#### Creating a PayloadCMS Collection
+
+You can create a new Collection called `Posts.ts` in `src/collections`.
+
+```astro title="src/collections"
+import { CollectionConfig } from "payload/types";
+
+const Posts: CollectionConfig = {
+  slug: "posts",
+  admin: {
+    useAsTitle: "title",
+  },
+  access: {
+    read: () => true,
+  },
+
+  fields: [
+    {
+      name: "title",
+      type: "text",
+      required: true,
+    },
+    {
+      name: "content",
+      type: "text",
+      required: true,
+    },
+    {
+      name: "slug",
+      type: "text",
+      required: true,
+    },
+  ],
+};
+
+export default Posts;
+```
+
+This creates a "Posts" collection which requires the following : `title`, `content`, `slug`. You will use this collection to manage your blog posts. The colllection is publicly available, so you shouldn't put any sensitive info inside.
+
+You then need to import and add it to the available collections in the `payload.config.ts` file.
+
+```astro title="src/payload.config.ts"
+import { buildConfig } from "payload/config";
+import path from "path";
+
+import Users from "./collections/Users";
+import Posts from "./collections/Posts";
+
+export default buildConfig({
+  serverURL: "http://localhost:3000",
+  admin: {
+    user: Users.slug,
+  },
+  collections: [Users, Posts],
+  typescript: {
+    outputFile: path.resolve(__dirname, "payload-types.ts"),
+  },
+  graphQL: {
+    schemaOutputFile: path.resolve(__dirname, "generated-schema.graphql"),
+  },
+});
+```
+
+If you then go into the PayloadCMS Dashboard, you should see a new collection called "Posts" appear next to the "Users" collection. Enter the "Posts" collection and create a new post. After saving it, you will notice the API URL appear in the bottom right corner.
```

Comment:
Just pointing out that I'm not sure where the "skippable steps" stop here. If someone chose a template (e.g. blog) would n original post have been created, so this step of creating a new post should also be skipped?

It also seems like the note about both services using port 3000 should not be in the "skipped" section... so I think *somewhere* here we might need a new subheading so it's more clear where a reader with a template should jump back in to the instructions?

(Note: I did make an earlier suggested edit that took out the instruction to "skip ahead", but then I started the next instruction with "If you didn't choose a template..." So the idea is still the same: we need to make it clear when the "skip/if you choose a template" steps stop so that people are clear about where to jump back in.)

Reply:
Excellent point here, most of these steps can be skipped if the blog template is selected. Personally, I used the UI to create new content

Reply:
>  Personally, I used the UI to create new content

I think we are talking about the same thing. Do you mean the PayloadCMS Dashboard when you say "UI"?

Reply:
Sorry for the confusion, yes I am talking about the Dashboard UI